### PR TITLE
Feature/channels page

### LINKED ISSUE #1350: Display total subscription count on the side bar
Currently I can't see number of channels i subscribed. it's simple ui tag,
places to show it:
- sidebar ?
- profiles > view profile > subscription list page

### LINKED ISSUE #1506: Offer optional menu item "Channels" instead of channel list in the UI
**Is your feature request related to a usage problem (not a bug)? Please describe.**
I subscribe to many channels and it would be useful to have a menu entry "Channels" instead of a long list in the UI.

**Describe the solution you'd like to see implemented**
- Add a seperate page for Subscribed Channels
- (optional) add a search bar for the channel page (would be very convenient)

**Describe alternatives you've considered**

-

**Screenshots**

-

**Additional context**

-

### LINKED ISSUE #2066: [Feature Request]: CHANNEL FILTER
### Guidelines

- [X] I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

Groups can only help so much for isolating a single channel in the subscriptions feed 

### Proposed Solution

I communicate better visually than in writing:

FreeTube_Channel_Filter_Needed_Feature

### Alternatives Considered

Having the option to switch between a list and/or grid layout would a weaker alternative

### Issue Labels

ease of use improvement

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by hockerschwan**
Title
---

**Pull Request Type**
- [x] Feature Implementation

**Related issue**
Closes #1350
Closes #1506
Closes #2066

**Description**
Subscribed channels list (similar to ft-profile-channel-list)

**Screenshots (if appropriate)**

**Testing (for code that is not small enough to be easily understandable)**
Manually adding/deleting channels

**Desktop (please complete the following information):**
 - OS: Windows 10
 - OS Version: 21H1
 - FreeTube version: 0.16.0

**Additional context**
Feedbacks appreciated ( spacing / size / wording etc.)
~~Reminder: rewrite ` thumbnailURL` when #1783 is implemented.~~

### CODE REVIEW

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
I think it better to move the channels tab icon to the top of the channels list on the sidebar because channels already has a dedicated section

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
I'll mark this with WIP because i'd like to see #1783 merged first

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> Reminder: rewrite thumbnailURL when #1783 is implemented.

@hockerschwan #1783 has been merged

**Comment by hockerschwan** [comment]
added update thumbnail and unsubscribe prompt

>I think it better to move the channels tab icon to the top of the channels list on the sidebar because channels already has a dedicated section

it looks ... out of place
z21

File: static/locales/en-US.yaml [hunk 1]
```diff
@@ -89,6 +89,15 @@ Subscriptions:
...
 More: More
+Channels:
+  Channels: Channels
+  Title: Channel List
+  Search bar placeholder: Search Channels
+  Count: $ channel(s) found.
+  Empty: Your channel list is currently empty.
+  Unsubscribe: Unsubscribe
+  Unsubscribed: $ has been removed from your subscriptions
+  Unsubscribe Prompt: Are you sure you want to unsubscribe "$"?
```
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [inline static/locales/en-US.yaml:100]: change to Are you sure you want to unsubscribe from "$"?

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — CHANGES_REQUESTED** [review]

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — CHANGES_REQUESTED** [review]
Last thing before approving. If list is empty hide the search bar. Just like it is implemented in the playlist and history page

**Comment by hockerschwan** [comment]
done

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]
lgtm!

**Review by GilgusMaximus — APPROVED** [review]

**Review by PikachuEXE — COMMENTED** [review]
Code review done
Give me some time to do local feature review...

**Review by PikachuEXE — APPROVED** [review]

**Comment by toby63** [comment]
Thx for implementing this, but could you add one more thing?
A way to optionally disable the channel list on the sidebar.

**Comment by absidue** [comment]
@toby63 That was already in FreeTube before this PR: Distraction Free Settings -> Hide Active Subscriptions

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@absidue maybe @toby63 means disabling the channels tab in the sidebar?

**Comment by toby63** [comment]
@absidue Thx, somehow I overlooked it, even though I was checking twice :sweat_smile:.

**Pull request merged**